### PR TITLE
Redo landing page and intro doc

### PR DESCRIPTION
**Pull request by nmdefries**
- Add more descriptive info to landing page
- Add citation and licensing info to landing page
- Move installation, setup, and basic usage from landing page to intro vignette
- Move signal discovery and example "other" endpoint calls from intro vignette to new signal discovery vignette
- Add geo info to intro vignette
- Add examples of geo specification to intro vignette
- Add examples of `as_of` specification to intro vignette
- Add stub versioned data vignette

Partially addresses https://github.com/cmu-delphi/epidatr/issues/212

### CODE REVIEW

File: vignettes/epidatr.Rmd [hunk 1]
```diff
@@ -11,379 +11,224 @@ vignette: >
...
-API docs: <https://cmu-delphi.github.io/delphi-epidata/api/ecdc_ili.html>
 
-```{r, eval = FALSE}
-pub_ecdc_ili(regions = "Armenia", epiweeks = 201840)
-```
+## Plotting
 
-#### KCDC ILI
+Because the output data is in a standard `tibble` format, we can easily plot
+it using `ggplot2`:
```
Comment (nmdefries) [inline vignettes/epidatr.Rmd:176]: Should we suggest any particular plotting libraries, especially for more complex plots like choropleths?

File: vignettes/epidatr.Rmd [hunk 2]
```diff
@@ -11,379 +11,224 @@ vignette: >
...
-Example queries with all the endpoint functions available in this package are given [below](#example-queries).
+To generate your key,
+[register for a pseudo-anonymous account](https://api.delphi.cmu.edu/epidata/admin/registration_form).
+See the `save_api_key()` function documentation for details on how to set up
+`epidatr` to use your API key.
 
-## Advanced Usage (Experimental)
+_Note_ that private endpoints (i.e. those prefixed with `pvt_`) require a
+separate key that needs to be passed as an argument. These endpoints require
+specific data use agreements to access.
```
Comment (nmdefries) [inline vignettes/epidatr.Rmd:57]: Need to include more info here. How can users get access to private endpoints? Do we have a page or contact email for that?

**Comment by dshemetov** [comment]
Are there parts you would like specific feedback on here? I've read so many variations on these docs that it's hard for me to focus and make helpful suggestions. If you don't need specific feedback, I'd be happy to just trust your taste in re-organizing this information.

File: vignettes/epidatr.Rmd (hunk 2 above)
Comment (dshemetov) [reply vignettes/epidatr.Rmd:57]: Never found that information anywhere. @brookslogan do you know?

File: vignettes/epidatr.Rmd (hunk 1 above)
Comment (dshemetov) [reply vignettes/epidatr.Rmd:176]: A choropleth example would be neat if there's a package we could use, but it's pretty low priority imo.

**Comment by nmdefries** [comment]
No specific feedback desired. Give as much or as little feedback as you want.

The changes here can be summarized as:

- give contextual, not usage, info on the landing page
- add more, shorter vignettes
- add more depth and specifics on usage

**Review by dshemetov — APPROVED** [review]
Trusting Nat's reorganization here. If there are typos, small edits needed, we can catch them after this is merged into dev and we work on other PRs for the next version.

File: vignettes/epidatr.Rmd (hunk 2 above)
Comment (brookslogan) [reply vignettes/epidatr.Rmd:59]: I don't.  Pre-COVID I don't think there was even a procedure, but I'm not sure if there is one now.

Context: Pre-COVID I think most of these we probably didn't even have permission to re-share.  Many were for internal use (e.g., feeding into ILI-Nearby) + potentially sharing back/visualizing to the providers or maybe very limited pre-agreed-upon third parties (and even this last possibility I'm not sure ever actually happened).  One that I know of (NoroSTAT) was private just for testing purposes; it requires revival work now though so I'm not sure there's a point in broadening access.

Comment (nmdefries) [reply vignettes/epidatr.Rmd:59]: Will look into this and address separately.

**Pull request merged**